frontend/src/pages/api/nodes.js

```diff
@@ -1,16 +1,54 @@
+import IPData from "ipdata";
 import { ExplorerApi } from "../../libraries/explorer-wamp";
 
+const ipdata = new IPData(
+  "c8b80db74e687b7471135f099c0332c36899f40df0a2716e7a66d57e"
```

Comment:
Just for future reference, NEVER commit code with real credentials (even if you believe that there is no harm can be made), https://medium.com/@selvaganesh93/what-happens-if-you-accidentally-commit-your-aws-access-token-to-public-github-be50d378b4c7

The very first moment you introduce some key, create a config for it immediately, and gitignore it.

Reply:
got it